canonical-websites/www.ubuntu.com
# Turn toc and links from divs to navs

## Done

- Turn toc and links from divs to navs so they do not appear printed pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally and see that it looks the same as live in your web browser at: 
- [/legal/short-terms/2015-06-24.html](http://0.0.0.0:8001/legal/short-terms/2015-06-24.html]
- [/legal/short-terms/2015-09-09.html](http://0.0.0.0:8001/legal/short-terms/2015-09-09.html]
- [/legal/short-terms/2016-05-20.html](http://0.0.0.0:8001/legal/short-terms/2016-05-20.html]
- [/legal/short-terms/2016-06-24.html](http://0.0.0.0:8001/legal/short-terms/2016-06-24.html]
- [/legal/short-terms/index.html](http://0.0.0.0:8001/legal/short-terms/index.html]
- [/legal/short-terms/ja.html](http://0.0.0.0:8001/legal/short-terms/ja.html]
- [/legal/ubuntu-advantage/service-description-ja/2017-02-09.html](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-02-09.html]
- [/legal/ubuntu-advantage/service-description-ja/index.html](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/index.html]
- [/legal/ubuntu-advantage/service-description.html](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description.html]
- [/legal/ubuntu-advantage/ua-terms.html](http://0.0.0.0:8011/legal/ubuntu-advantage/ua-terms.html)

## Issue / Card

Fixes #2516
